jekyll/_cci2/creating-orbs.md

```diff
@@ -21,122 +21,56 @@ After authoring your orb, you can publish it with a [semantically versioned]({{s
 
 ![Orb Publishing Process]({{ site.baseurl }}/assets/img/docs/orb-publishing-process.png)
```

Comment:
@KyleTryon Do you think we need to reference private orbs everywhere? the line above says if you publish your orb will be in the registry 🤔

Reply:
You will be able to view your orb on the registry even if it is private if:
1. You are logged in and authenticated to the correct account
2. You know the exact URL or name of the orb.

You will not be able to search for the orb via the search box.